addons/l10n_co_pos/static/src/js/pos.js

```diff
@@ -2,31 +2,33 @@ odoo.define('l10n_co_pos.pos', function (require) {
 "use strict";
 
 var models = require('point_of_sale.models');
+const Registries = require('point_of_sale.Registries');
 
-models.PosModel = models.PosModel.extend({
-    is_colombian_country: function () {
+const L10nCoPosGlobalState = (PosGlobalState) => class L10nCoPosGlobalState extends PosGlobalState {
```

Comment:
can you explain me the syntax of this I don't really understand it. Why do you need "const L10nCoPosGlobalState = (PosGlobalState) =>"?

Reply:
forget this one, I get it